### PR TITLE
use resolutions to lock intl-format-cache version

### PR DESCRIPTION
**Pull request by mjuniper**
We suddenly started getting build errors in travis (though not locally, even with a clean checkout). They seem to be related to recent changes in [intl-format-cache](https://github.com/formatjs/formatjs/blob/master/packages/intl-format-cache/CHANGELOG.md) which is used by the version of ember-intl we are on. I tried upgrading ember-intl to the latest 2.X, and I also tried using node 10. None of that worked (but I kept those changes anyway because I think they are good).

So the consequential change that makes the build green is to add `resolutions` to package.json and specify the last version of `intl-format-cache` that works.

An alternative that would probably work would be to upgrade to ember-intl 3.X or 4.X but I am not prepared to tackle that right now.

The question/concern I have is: why was this only a problem for travis and why did we not see it in jenkins/opendata-ui?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5030,7 +5030,7 @@ insert-module-globals@~6.0.0:
     through "~2.3.4"
     xtend "^3.0.0"
 
-intl-format-cache@^2.0.5:
+intl-format-cache@2.1.0, intl-format-cache@^2.0.5:
```
Comment (dmfenton) [inline yarn.lock]: it was already resolved to 2.1.0, how does this change anything?

Comment (dmfenton) [reply yarn.lock]: does this cause in changes in the yarn.lock's of consuming packages?

Comment (mjuniper) [reply yarn.lock]: Ha! I have no idea! But we consistently got failed builds last night and now we do not. Lemme try something.

Comment (mjuniper) [reply yarn.lock]: @dmfenton I just removed the `resolutions` and pushed that commit and the build failed. Then I re-added the `resolutions` and the build succeeded. I also triggered a build on master and it failed. 

So I cannot explain exactly whey this works but I am confident that it does.

Comment (tomwayson) [reply yarn.lock]: > does this cause in changes in the yarn.lock's of consuming packages?

I would not expect it too

**Pull request merged**